### PR TITLE
 [JA] Fix spelling and punctuation marks

### PR DESCRIPTION
**Pull request by kawaemon**
- Added punctuation marks.
- Corrected broken markdowns.
- Corrected some of writing errors.

### CODE REVIEW

File: wiki/FAQ/ja.md [hunk 1]
```diff
@@ -3,18 +3,18 @@
 Osu!とは
 ========
 
-このゲームのメインとなるモードで、カーソルを動かしサークル、スライダー、スピナーの3つをタイミングよく叩くモードです
-叩くタイミングの正確さによって300、100、50、missと4つの判定が出ます
-また[太鼓モードや](/wiki/Game_Modes/osu!taiko)[CTBとは別々にスコア管理がされます](/wiki/Game_Modes/osu!catch)
+このゲームのメインとなるモードで、カーソルを動かしサークル、スライダー、スピナーの3つをタイミングよく叩くモードです。
+叩くタイミングの正確さによって300、100、50、missと4つの判定が出ます。
+また[太鼓モードや](/wiki/Game_Modes/osu!taiko)[CTBとは別々にスコア管理がされます](/wiki/Game_Modes/osu!catch)。
```
Comment (cl8n) [inline wiki/FAQ/ja.md]:
```suggestion
また[太鼓モード](/wiki/Game_Modes/osu!taiko)や[osu!catch](/wiki/Game_Modes/osu!catch)とは別々にスコア管理がされます。
```
if there is a Japanese word for "osu!catch" you can use that instead

File: wiki/FAQ/ja.md [hunk 2]
```diff
@@ -96,83 +96,86 @@ Cよりも下の条件でクリア
...
 
 ### ダウンロード速度が上がる
 
 プレイ方法
 ==========
 
-メインメニューのSOLOを選択すればプレイできます
-ですが、一部の譜面は強制的に[太鼓モードまたは](/wiki/Game_Modes/osu!taiko)[CTBモードにされるものもあるので注意してください](/wiki/Game_Modes/osu!catch)
+メインメニューのSOLOを選択すればプレイできます。
+ですが、一部の譜面は強制的に[太鼓モードまたは](/wiki/Game_Modes/osu!taiko)[CTBモードにされるものもあるので注意してください](/wiki/Game_Modes/osu!catch)。
```
Comment (cl8n) [inline wiki/FAQ/ja.md]:
```suggestion
ですが、一部の譜面は強制的に[太鼓モード](/wiki/Game_Modes/osu!taiko)または[osu!catch](/wiki/Game_Modes/osu!catch)にされるものもあるので注意してください。
```
let's drop the "モード" when writing game mode titles. it makes sense for "太鼓モード", but "CTBモード" or "osu!catchモード" are just redundant

**Comment by kawaemon** [comment]
Thank you clayton for suggesting!
Committed it.

File: wiki/FAQ/ja.md (hunk 1 above)
Comment (Joehuu) [reply wiki/FAQ/ja.md]: I don't think game modes should be translated.

Comment (kawaemon) [reply wiki/FAQ/ja.md]: I agree.
I'll change them.

**Comment by kawaemon** [comment]
- Corrected some of writing errors
- Added about osu!mania ( but it haven't translated yet... should I delete it? )

And I'm sorry for dismissing stable review. I didn't notice these misses..

File: wiki/FAQ/ja.md [hunk 3]
```diff
@@ -96,83 +96,86 @@ Cよりも下の条件でクリア
...
-### ダウンロード速度が上がる
+### 譜面ダウンロード速度が上がる
 
 プレイ方法
 ==========
 
-メインメニューのSOLOを選択すればプレイできます
-ですが、一部の譜面は強制的に[太鼓モードまたは](/wiki/Game_Modes/osu!taiko)[CTBモードにされるものもあるので注意してください](/wiki/Game_Modes/osu!catch)
+メインメニューのSOLOを選択すればプレイできます。
+ですが、一部の譜面は強制的に[osu!taiko](/wiki/Game_Modes/osu!taiko)、[osu!catch](/wiki/Game_Modes/osu!catch)または[osu!mania](/wiki/Game_Modes/osu!mania)に変更されるので注意してください。
```
Comment (lolita-is-godddd) [inline wiki/FAQ/ja.md]: It should `譜面の中には[osu!taiko](/wiki/Game_Modes/osu!taiko)、[osu!catch](/wiki/Game_Modes/osu!catch)または[osu!mania](/wiki/Game_Modes/osu!mania)専用のものがあります。` to clear meaning

Comment (lolita-is-godddd) [reply wiki/FAQ/ja.md]: Wrong suggestion... the comment one is right

Comment (kawaemon) [reply wiki/FAQ/ja.md]: Fixed and committed.
Thank you for suggesting!

**Pull request merged**